components/post_view/post_list.jsx

```diff
@@ -228,12 +270,20 @@ export default class PostList extends React.PureComponent {
     }
 
     loadMorePosts = async () => {
-        const oldestPostId = this.getOldestVisiblePostId();
+        const oldestPostId = PostList.getOldestVisiblePostId(this.state.postListIds);
 
         if (!oldestPostId) {
             // loadMorePosts shouldn't be called if we don't already have posts
             return;
         }
+        if (this.state.postListIds.length - 1 !== this.props.postListIds.length) {
```

Comment:
Is the `- 1` to remove the extra row that contains things like the intro message?

Reply:
Could you add a comment explaining what this block does? It's not really obvious that it's showing more posts that were already loaded.